taskwiki/regexp.py

```diff
@@ -46,8 +46,12 @@
     re.compile(
         r'^'                         # Starts at the begging of the line
         r'(?P<header_start>[=]+)'    # Heading begging
-        r'(?P<name>[^=\|\[\{]*)'     # Name of the viewport, all before the | sign
-                                     # Cannot include '[', '=', '|', and '{'
+
+        r'(?P<name>([^=\|\[\{]*)|(\s*\{\{.*}}\s*)|(\s*\[\[.*]]\s*))'
```

Comment:
This looks like it only parses headers that start and end with a vimwiki link, but not headers that contain the link somewhere in the middle. Is that intended?

Reply:
Hm, that was the use case I had in mind, do we need links as part only of the header name?

Reply:
I'd agree with @liskin here, if we're addressing this, we might as well address the general case, i.e.:

    ## Tasks related to [website](https//myproject.com) | project:website

should also work.

Reply:
Done.